<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeepvalWidget</class>
 <widget class="QWidget" name="DeepvalWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Check tree depth</string>
  </property>
  <property name="windowIcon">
   <iconset theme="view-sort-ascending">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <property name="rowWrapPolicy">
    <enum>QFormLayout::DontWrapRows</enum>
   </property>
   <property name="labelAlignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="formAlignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="minLabel">
     <property name="text">
      <string>Minimum tree depth</string>
     </property>
     <property name="buddy">
      <cstring>minSpin</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="minSpin">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="maxLabel">
     <property name="text">
      <string>Maximum tree depth</string>
     </property>
     <property name="buddy">
      <cstring>maxSpin</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="maxSpin">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="allowLabel">
     <property name="text">
      <string>Allow files from</string>
     </property>
     <property name="buddy">
      <cstring>allowSpin</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="allowSpin">
     <property name="specialValueText">
      <string comment="level">Any level</string>
     </property>
     <property name="suffix">
      <string> level</string>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>minSpin</tabstop>
  <tabstop>maxSpin</tabstop>
  <tabstop>allowSpin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
